featuretools/computational_backends/pandas_backend.py

```diff
@@ -277,7 +280,9 @@ def generate_default_df(self, instance_ids, extra_columns=None):
         return default_df
 
     def _feature_type_handler(self, f):
-        if isinstance(f, TransformFeature):
+        if isinstance(f, GroupByTransformFeature):
```

Comment:
might be worth leaving a comment saying the `GroupByTransformFeature` has to come first since it is a subclass of TransformFeature, unless we change it as suggested above